solidity/solc-update-0.8.15-to-0.8.25.md

```diff
@@ -0,0 +1,98 @@
+- Relax restrictions on initialization of immutable variables. Reads and writes may now happen at any point at construction time outside of functions and modifiers. Explicit initialization is no longer mandatory (>= 0.8.21)
+- Support for the EVM Version "Cancun" (>= 0.8.24): Introduces global `block.blobbasefee`, `blobhash(uint)`, `blobbasefee()`, `blobhash()`, `mcopy()`, `tload()` and `tstore()`.
+
+## Notable Features
+
+Highlight any potentially notable features that require special attention during the review
+process. Features should be considered "notable" if they add significant new functionality to
+Solidity or required significant changes to Solidity to be supported.
+
+- Deprecate support for "homestead", "tangerineWhistle", "spuriousDragon" and "byzantium" EVM versions: This might need special attention if any contracts are required to be compiled with any of the deprecated EVM versions. This won't be possible with the v0.8.25.
+
+## Notable Bug Fixes
+
+Highlight any potentially notable bug fixes that require special attention during the review
+process. Bug fixes should be considered "notable" if they are classified at or above a "medium" by
+the Solidity team.
+
+- [Head Overflow Bug in Calldata Tuple ABI-Reencoding](https://soliditylang.org/blog/2022/08/08/calldata-tuple-reencoding-head-overflow-bug/) (Bug fixed in v0.8.16)
+- [Storage write removal before conditional termination](https://soliditylang.org/blog/2022/09/08/storage-write-removal-before-conditional-termination/) (Bug fixed in v0.8.17)
+- [FullInliner Non-Expression-Split Argument Evaluation Order Bug](https://soliditylang.org/blog/2023/07/19/full-inliner-non-expression-split-argument-evaluation-order-bug/) (Bug fixed in v0.8.21)
+
+## Changelog Links
+
+- [Solidity 0.8.16 Release Announcement](https://soliditylang.org/blog/2022/08/08/solidity-0.8.16-release-announcement/)
+- [Solidity 0.8.17 Release Announcement](https://soliditylang.org/blog/2022/09/08/solidity-0.8.17-release-announcement/)
+- [Solidity 0.8.18 Release Announcement](https://soliditylang.org/blog/2023/02/01/solidity-0.8.18-release-announcement/)
+- [Solidity 0.8.19 Release Announcement](https://soliditylang.org/blog/2023/02/22/solidity-0.8.19-release-announcement/)
+- [Solidity 0.8.20 Release Announcement](https://soliditylang.org/blog/2023/05/10/solidity-0.8.20-release-announcement/)
+- [Solidity 0.8.21 Release Announcement](https://soliditylang.org/blog/2023/07/19/solidity-0.8.21-release-announcement/)
+- [Solidity 0.8.22 Release Announcement](https://soliditylang.org/blog/2023/10/25/solidity-0.8.22-release-announcement/)
+- [Solidity 0.8.23 Release Announcement](https://soliditylang.org/blog/2023/11/08/solidity-0.8.23-release-announcement/)
+- [Solidity 0.8.24 Release Announcement](https://soliditylang.org/blog/2024/01/26/solidity-0.8.24-release-announcement/)
+- [Solidity 0.8.25 Release Announcement](https://soliditylang.org/blog/2024/03/14/solidity-0.8.25-release-announcement/)
+
+## Action Items
```

Comment:
Looks like this is from the FMA template. Do we want to add this to the solc template as well? I think it might be a good idea as it helps us sanity check we don't e.g. exceed code size limits when changing versions. Any thoughts @smartcontracts @maurelian